blogposts/2021/release-3.29.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Sourcegraph 3.29 release"
+publishDate: 2021-06-18T10:00-07:00
+description: "The Sourcegraph 3.29 release introduces improved search results ranking, makes it easier to create Code Insights, and includes support for bulk actions with Batch Changes."
+tags: [blog, release]
+slug: "release/3.29"
+published: false
+changelogItems:
+  - description: "Code Insights queries can now run concurrently up to a limit set by the `insights.query.worker.concurrency` site config."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21219
+    category: Code Insights
+  - description: "Code Insights workers now support a rate limit for query execution and historical data frame analysis using the `insights.query.worker.rateLimit` and `insights.historical.worker.rateLimit` site configurations."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21533
+    category: 
+  - description: "The GraphQL `Site` `SettingsSubject` type now has an `allowSiteSettingsEdits` field to allow clients to determine whether the instance uses the `GLOBAL_SETTINGS_FILE` environment variable."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21827
+    category: API
+  - description: "Code Insights creation UI now has auto-save logic and clear all fields functionality"
```

Comment:
```suggestion
  - description: "Code Insights creation UI now has autosave logic and clear all fields functionality."
```
- Making consistent with use of autosave elsewhere
- Let's standardise on using periods at the end of these